### PR TITLE
Added DLP region fix

### PR DESCRIPTION
**Pull request by ghost**
I know its discontinued, but we should merge this.. 

P.S. Can I further the development since its discontinued? It is better than Luma3DS + Plus I have lots of ideas 😄

### CODE REVIEW

**Comment by ghost** [comment]
Never mind.. You already did.. My mistake..

**Comment by sykhro** [comment]
If it's discontinued, you shouldn't bother opening a PR at all.

**Comment by chaoskagami** [comment]
Indeed, I already did this.

I'm tempted to mark the repo archived, but I think I'll refrain from that.

As for whether you can fork and make changes: This project is licensed under the GPLv3 mostly, with some parts under other licenses.

In case the above doesn't explain: Open source licenses don't require "asking for permission."

If anyone has told you otherwise, they have attached another condition to their licensing which contradicts the terms of almost every single FOSS license. And/or they don't understand the terms of their own license choice.

tl;dr feel free to fork, it's GPL'd.

**Comment by gnmmarechal** [comment]
I didn't even know GitHub supported merging from somebody else's repo. Actually, I don't even remember where that code came from.

**Comment by ToadillerYT2** [comment]
LOL Sorry guys.. my bad...

Anyhow, I was wondering....

Would it be all right if I furthered the development of Corbenik? I am interested in building upon CFW's right now.

**Comment by ghost** [comment]
LOL Sorry my bad..

**Comment by chaoskagami** [comment]
@gnmmarechal - Technically speaking, PRs aren't even needed to merge commits from someone else's tree. They're merely a convenience, and some big name projects (e.g. Linux) still do stuff the old fashioned way.

Say a person had a change I wanted in their master tree:

```
git add remote other-person https://github.com/<person>/<repo>
git fetch other-person
```

and then you can pick one of these:

```
git merge master
```

```
git cherry-pick <commit hash>
```

or one of N other ways using `git rebase` or other commands. Long and short PRs are just a simplification of one of many ways to import code changes.

**Comment by gnmmarechal** [comment]
Yeah, I just didn't know you could to it from somebody else's repo. Then again, I'd never tried. I barely ever used PRs at all.
